config/locales/en.yml

```diff
@@ -343,7 +343,7 @@ en:
               blank: Choose who you are trying to get consent from
             reason_for_refusal:
               inclusion: Choose a reason
-            reason_for_refusal_notes:
+            notes:
               blank: Enter details for refusing
```

Comment:
I think the `blank` error messages need to be more general now?

Reply:
Good point, I've amended the commit to go with "Enter notes" which I think matches our other error messages.